cs2_movement_reverse_engineering.cpp

```diff
@@ -4540,27 +4602,36 @@ class CCSPlayerPawnBase : public CBasePlayerPawn
 	int32_t m_nSurvivalTeamNumber; 	// 0x14ec
 	// MNetworkEnable
 	// MNetworkBitCount "32"
-	Vector m_aimPunchAngle; 	// 0x14f0
+	QAngle m_aimPunchAngle; // 0x14f0
 	// MNetworkEnable
 	// MNetworkBitCount "32"
-	Vector m_aimPunchAngleVel; 	// 0x14fc
+	QAngle m_aimPunchAngleVel; // 0x14fc
 	// MNetworkEnable
 	int32_t m_aimPunchTickBase; 	// 0x1508
 	// MNetworkEnable
 	float m_aimPunchTickFraction; 	// 0x150c
-	CUtlVector m_aimPunchCache; 	// 0x1510 CUtlVector< QAngle >
+	CUtlVector m_aimPunchCache; // 0x1510
 	bool m_bHasDeathInfo; 	// 0x1528
+
+	uint8_t __pad1529[0x3]; // 0x1529
+
 	float m_flDeathInfoTime; 	// 0x152c
 	Vector m_vecDeathInfoOrigin; 	// 0x1530
 	// MNetworkEnable
 	bool m_bKilledByHeadshot; 	// 0x153c
+
+	uint8_t __pad153d[0x3]; // 0x153d
+
 	int32_t m_LastHitBox; 	// 0x1540
 	int32_t m_LastHealth; 	// 0x1544
 	float m_flLastCollisionCeiling; 	// 0x1548
 	float m_flLastCollisionCeilingChangeTime; 	// 0x154c
 	CUtlString m_strVOPrefix; 	// 0x1550
 	float m_flLandseconds; 	// 0x1558
-	void *m_pBot; 	// 0x1560 CCSBot *
+
+	uint8_t __pad155c[0x4]; // 0x155c
+
+	CCSBot* m_pBot; // 0x1560
```

Comment:
CCSBot class doesn't exist in this file, either make it a void * with a comment as to what the original type was (i'd prefer this cos i don't care about bots :) ), or add the CCSBot classes.